src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -112,9 +114,18 @@
         <div class="float-right">
             <form asp-resource="@Model.Review" class="form-inline float-left ml-2" asp-page-handler="ToggleApproval" method="post" asp-requirement="@ApproverRequirement.Instance">
                 <input type="hidden" name="revisionId" value="@Model.Revision.RevisionId" />
-                <input type="submit" class="btn btn-sm btn-secondary" value="@(Model.Revision.Approvers.Contains(User.GetGitHubLogin()) ? "Revert Approval" : "Approve")" />
+                @if (Model.DiffRevision == null || Model.DiffRevision.Approvers.Count > 0)

```

Comment:
Should we add an IsApproved property to the revision. We do `revision.Approvers.Count() > 0` in so many places.